### PR TITLE
School Library: Refactor Your Code

### PR DESCRIPTION
**Pull request by iamchristianani**
## Refactor Your Code

> In this Pull Request, @leehaney254 and I were able to complete the following:

- Refactored `main.rb` so that it starts the code and asks the user for the initial choice of options.
- Create a `menu.rb` file to separate the method calls from the method calling the app.
- Fills the UI accordingly.
- Fix the Linter errors.

### CODE REVIEW

File: app.rb [hunk 1]
```diff
@@ -36,9 +36,10 @@ def list_people
   def create_person
     print 'Do you want to create a student (1) or a teacher (2)? [Input the number]: '
     number = gets.chomp.to_i
```
Comment (youmari) [inline app.rb:38]: - [ ] for all the methods that create `books, students, teacher, rental` please make sure to separate the code that `requests for input and input validation` like these in the picture
Screenshot 2023-03-13 at 16 08 46
into their own functions (methods) and then use them inside create_student, create_book... etc
that way you will ensure that you're respecting the rule of the single responsibility principle

**Review by youmari — CHANGES_REQUESTED** [review]
Hi @iamchristianani @leehaney254, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights
✔️ Great commit messages
✔️ PRO PR
✔️ No linter error 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: app.rb (hunk 1 above)
Comment (iamchristianani) [reply app.rb:38]: Code has been fixed as requested

**Review by m05e5 — APPROVED** [review]
**Hi @iamchristianani and @leehaney254 
You did great. Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
CleverScornfulBuffalo-size_restricted

## Highlights

✔️ Clean and well-organized code
✔️ No Linters error
✔️ Good PR description

## Optional suggestions


_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

_**m05e5**_

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._**

**Pull request merged**